Library/ViewModels/SettingsNotificationsViewModel.swift

```diff
@@ -1,21 +1,25 @@
 import Foundation
 import KsApi
+import Library
 import Prelude
 import ReactiveSwift
 import ReactiveExtensions
 import Result
 
 public protocol SettingsNotificationsViewModelInputs {
   func didSelectRow(cellType: SettingsNotificationCellType)
+  func didSelectEmailFrequency(frequency: EmailFrequency)
   func failedToUpdateUser(error: String)
+  func showPickerView(show: Bool)
```

Comment:
This also reads like an output, so it should probably be more like `func didTapFrequencyPickerButton()` which is really the event that occurred, also seeing as that always passes in true then this can just be mapped to `true`/`false` on [line 86](https://github.com/kickstarter/ios-oss/pull/401/files#diff-5201ce86be2c9258634f949a836fff31R86).